connector/connect/client/jvm/pom.xml

```diff
@@ -44,11 +44,18 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-sql-api_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
+      <exclusions>
```

Comment:
@HyukjinKwon This should solve the problem, but I also want to try some other approaches.

Reply:
👌